test/index.js

```diff
@@ -478,27 +489,39 @@ test('NSpell()', function (t) {
     st.deepEqual(
       us.suggest('dont'),
       [
-        'dent',
         'cont',
-        'font',
-        'wont',
+        'dent',
         'dint',
+        'done',
+        'dong',
+        'font',
```

Comment:
For "dont" we now actually have 5 equally-weighted keyboard edits instead of the previous sole edit of "dent".